tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -1609,6 +1609,22 @@ struct strange_view5 : strange_view4, ranges::view_interface<strange_view5> {
 STATIC_ASSERT(!std::is_base_of_v<std::ranges::view_base, ranges::view_interface<strange_view4>>);
 STATIC_ASSERT(!std::is_base_of_v<std::ranges::view_base, ranges::view_interface<strange_view5>>);
 
+// Verify that enable_view<T&> or enable_view<T&&> is never true
+STATIC_ASSERT(!ranges::enable_view<strange_view4&>);
+STATIC_ASSERT(!ranges::enable_view<const strange_view4&>);
+STATIC_ASSERT(!ranges::enable_view<strange_view4&&>);
+STATIC_ASSERT(!ranges::enable_view<const strange_view4&&>);
+
+// Verify that the derived-from-view_interface mechanism can handle uses of incomplete types whenever possible
+struct incomplet;

```

Comment:
Why the mis-spelling of `incomplete` here?

Reply:
"Note: this is an early draft. It’s known to be incomplet and incorrekt, and it has lots of bad forma<sub>t</sub>ti<sup>n</sup>g."

Reply:
It's a little bit silly but I think it's acceptable.

Reply:
> 

This is ~incorrekt~ incorrect but consistent with the C++ Working Draft.😺